### PR TITLE
Feature/confirmation of booking

### LINKED ISSUE #7: Confirmation of booking
Implement confirmation of booking by host

### PR DESCRIPTION
**Pull request by rajtarovN**
Closes #7

### CODE REVIEW

File: reservation-service/src/main/java/com/reservation/reservationservice/dtos/ReservationDTO.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+import java.util.Date;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Data
+public class ReservationDTO {
+    private String Id;
+    private Date dateFrom; //pitaj da li da stavim local
```
Comment (RajnovicTeodora) [inline reservation-service/src/main/java/com/reservation/reservationservice/dtos/ReservationDTO.java:19]: Mislim da da, po potrebi izmijeniti kasnije

File: reservation-service/src/main/java/com/reservation/reservationservice/service/RequestService.java [hunk 2]
```diff
@@ -1,7 +1,82 @@
...
+    @Autowired
+    private AccomodationRepository accomodationRepository;
+    @Autowired
+    private ReservationRepository reservationRepository;
+
+    @Autowired
+    private GuestRepository guestRepository;
+    public ReservationDTO approveRequest(String id) throws BadRequestException {
+
+        Optional<Request> request =  this.requestRepository.findById(id);
```
Comment (RajnovicTeodora) [inline reservation-service/src/main/java/com/reservation/reservationservice/service/RequestService.java:34]: Why not put request = this.requestRepository.findById(id).get() and maybe orElseThrow(EntityNotFound::new)

**Review by RajnovicTeodora — APPROVED** [review]

File: reservation-service/src/main/java/com/reservation/reservationservice/dtos/ReservationDTO.java (hunk 1 above)
Comment (rajtarovN) [reply reservation-service/src/main/java/com/reservation/reservationservice/dtos/ReservationDTO.java:19]: okej

File: reservation-service/src/main/java/com/reservation/reservationservice/service/RequestService.java (hunk 2 above)
Comment (rajtarovN) [reply reservation-service/src/main/java/com/reservation/reservationservice/service/RequestService.java]: done

**Pull request merged**